oap-server/server-library/library-client/src/main/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ElasticSearchClient.java

```diff
@@ -210,7 +210,8 @@ public void afterBulk(long executionId, BulkRequest request, Throwable failure)
         return BulkProcessor.builder(client::bulkAsync, listener)
             .setBulkActions(bulkActions)
             .setBulkSize(new ByteSizeValue(bulkSize, ByteSizeUnit.MB))
-            .setFlushInterval(TimeValue.timeValueSeconds(flushInterval))
+            // modified by zhaoze, if set auto flush, there will cause deadlock with bulk commit.
```

Comment:
You can't set this comment in the repo, remove it

Reply:
Make comment without yout name, just why.

Reply:
OK, i got it